server/config/conditions.ts

```diff
@@ -449,41 +533,106 @@ export default {
-    //   requiresInput: true,
-    // },
-    // {
-    //   code: '86e6f2a9-bb60-40f8-9ac4-310ebc72ac2f',
-    //   category: 'Electronic monitoring',
-    //   text: 'You must stay at [APPROVED ADDRESS] between 5pm and midnight every day until your electronic tag is installed unless otherwise authorised by your supervising officer.',
-    //   requiresInput: true,
-    // },
-    // {
-    //   code: '599bdcae-d545-461c-b1a9-02cb3d4ba268',
-    //   category: 'Electronic monitoring',
-    //   text: 'You are subject to alcohol monitoring. Your alcohol intake will be electronically monitoring for a period of [INSERT TIMEFRAME AND END DATE], and you may not consume units of alcohol, unless otherwise permitted by your supervising officer.',
-    //   requiresInput: true,
-    // },
+    {
+      code: 'fd129172-bdd3-4d97-a4a0-efd7b47a49d4',
+      category: 'Electronic monitoring',
+      text: 'Allow person(s) as designated by your supervising officer to install an electronic monitoring tag on you and access to install any associated equipment in your property, and for the purpose of ensuring that equipment is functioning correctly. You must not damage or tamper with these devices and ensure that the tag is charged, and report to your supervising officer and the EM provider immediately if the tag or the associated equipment are not working correctly. This will be for the purpose of monitoring your [INSERT TYPES OF CONDITIONS TO BE ELECTRONICALLY MONITORED HERE] licence condition(s) unless otherwise authorised by your supervising officer.',
+      requiresInput: true,
+      inputs: [
+        {
+          type: InputTypes.CHECK,
+          label: 'Select all that apply',
+          name: 'electronicMonitoringTypes',
+          options: [
+            {
+              value: 'exclusion zone',
+            },
+            {
+              value: 'curfew',
+            },
+            {
+              value: 'location monitoring',
+            },
+            {
+              value: 'attendance',
+            },
+          ],
+        },
+      ],
+      type: ElectronicMonitoringTypes,
+    },
+    {
+      code: '524f2fd6-ad53-47dd-8edc-2161d3dd2ed4',
+      category: 'Electronic monitoring',
+      text: 'You will be subject to trail monitoring. Your whereabouts will be electronically monitored by GPS Satellite Tagging, ending on [INSERT END DATE], and you must cooperate with the monitoring as directed by your supervising officer unless otherwise authorised by your supervising officer.',
+      requiresInput: true,
+      inputs: [
+        {
+          type: InputTypes.TEXT,
+          label: 'Enter the timeframe',
+          name: 'timeframe',
+        },
+        {
+          type: InputTypes.DATE,
+          name: 'endDate',
+        },
+      ],
+      type: ElectronicMonitoringPeriod,
+    },
```

Comment:
This condition has only one input field, but two values? Is this an either-or scenario? If a time-frame, the sentence won't make sense!

Reply:
Oh! good spot, I hadn't realised the condition text had changed content. The inputs are correct, the text is not